app/controllers/concerns/fraud_review_concern.rb

```diff
@@ -28,7 +28,8 @@ def handle_fraud_rejection
 
   def in_person_prevent_fraud_redirection?
     IdentityConfig.store.in_person_proofing_enforce_tmx &&
-      current_user.ipp_enrollment_status_not_passed?
+      current_user.ipp_enrollment_status_not_passed? &&
+      (fraud_review_pending? || fraud_rejection?)
```

Comment:
Undoing previous change around fraud rejection. See this [slack thread for justification](https://gsa-tts.slack.com/archives/C03FA4VBN76/p1710973042439869?thread_ts=1710972257.115159&cid=C03FA4VBN76)

Checking for fraud_review_pending and fraud_rejection because this is called before both fraud redirection cases.

Reply:
Jack- can you explain again why the additional check ((fraud_review_pending? || fraud_rejection?)) is needed?

Reply:
I will add a comment above this method to help clarify what's going on here.

Reply:
@gina-yamada @n1zyy @KeithNava Comment added [here](https://github.com/18F/identity-idp/pull/10391/commits/608f670559b67643fa6dc80684be339f29d7d107)